cmd/bootstrapper/initialize_kubelet.go

```diff
@@ -51,6 +57,8 @@ func init() {
 	initializeKubeletCmd.PersistentFlags().StringVar(&initializeKubeletOpts.nodeIP, "node-ip", "",
 		"nodeIP is the IP that should be used as the node object's IP. "+
 			"If unset, kubelet will determine the IP itself.")
+	initializeKubeletCmd.PersistentFlags().StringVar(&initializeKubeletOpts.clusterDNSIP, "cluster-dns-ip", "",
+		"IP address used to configure the kubelet with a specific clusterDNS IP")
```

Comment:
Would prefer this to describe the end results for the user. So something like:
`The DNS server IP which will be used by all containers, instead of the host's DNS servers`

Paraphrasing from https://kubernetes.io/docs/reference/config-api/kubelet-config.v1beta1/